src/main/kotlin/token/ERC20TokenRegistrationConfig.kt

```diff
@@ -4,12 +4,16 @@ import config.IrohaConfig
 import config.IrohaCredentialConfig
 
 interface ERC20TokenRegistrationConfig {
+
+    /** Iroha credentials */
+    val irohaCredential: IrohaCredentialConfig
```

Comment:
seems a bit confusing for me. can we make irohaCredential look more meaningful? like for example tokenCreatorCredential

Reply:
It is a credentials to sign transactions sent by ERC20TokenRegistration. Previously it was creator and keypair.

Reply:
can we name it like 'tokenRegistrationCredentials'? or it has no sense?

Reply:
I think it has no sense since it is in TokenRegistration.